core/server/models/post.js

```diff
@@ -517,6 +517,51 @@ Post = ghostBookshelf.Model.extend({
         }
 
         return options.context && options.context.public ? 'page:false' : 'page:false+status:published';
+    },
+
+    /**
+     * You can pass an extra `status=VALUES` or "staticPages" field.
+     * Long-Term: We should deprecate these short cuts and force users to use the filter param.
+     */
+    extraFilters: function extraFilters(options) {
+        if (!options.staticPages && !options.status) {
+            return null;
+        }
+
+        let filter = null;
+
+        // CASE: "staticPages" is passed
+        if (options.staticPages && options.staticPages !== 'all') {
```

Comment:
Do we still need to support staticPages field? We ignore it in public apis, e.g.: https://github.com/TryGhost/Ghost/blob/9fd9186557684e30afc910ebe5c137f52adab268/core/test/functional/api/v0.1/public_api_spec.js#L435-L457

The only use seems to be within integration tests for model e.g: https://github.com/TryGhost/Ghost/blob/a7b00294713c6e0f51d2f77ce24d38df900cf021/core/test/integration/model/model_posts_spec.js#L271

Reply:
Yes we have to. This is a field for the admin/private api. We should get rid of both short cuts at some point.

Reply:
See v0.1 posts ctrl

Reply:
Ok :+1: . Got it, was looking for the explanation in tests instead of going through source code.